.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
It would be better to add the `time: "21:00"` option under `schedule`, and then we can maintain dependency upgrades at a fixed time every day, which is more convenient than random time.

Reply:
I don't think we need to change this, since the default is already fixed at `5:00 UTC` according to https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates#scheduletime

Reply:
Cool :+1: